README.md

```diff
@@ -2,27 +2,236 @@
+- **`llm_model` ->** Model name/version to use with the provider API, for example: `google/gemini-2.0-flash-exp:free`. Set to 'webhook' to proxy request as POST to `llm_api_url`, and sending `llm_key` as the `token` key of the json body.
+
+## Creating an Alexa Skill
+
+To use this template, you need to at least have an account setup in the [Alexa Developer Console](https://developer.amazon.com/alexa).
+
+**There's three ways you can use this template in a skill:**
+- **Automated ->** Using the `Makefile` to create and manage a new or imported Alexa Hosted Skill project
+- **Manual ->** in the the `Alexa Developer Console` itself, by uploading a build package
+- **Advanced ->** Using the `ask CLI` to create and manage a new AWS-hosted or Self-hosted skill project using this repository as template.
+
+### Automated - Using the Makefile (Alexa Hosted Skills Management)
+
+>*ℹ️ This is the recommended way to create a new Alexa Skill using this template. It leverages the Ask CLI to create a new project and deploy it to your Alexa Developer Console. You can have multiple targets and deploy the template to different skills.*
+
+This method supports version control, testing, and debugging, and integrates with the Alexa Developer Console seamlessly.
+
+>*⚠️ Make sure you have the `ask` CLI installed and configured with your Amazon Developer account before running this command. If not, install it by running `npm install -g ask-cli` and configure it by running `ask configure`.*
+
+#### Create a new Alexa Skill
+
+Run the following command in your terminal:
+
+```bash
+make new
+```
+
+And follow the wizard to create a new Alexa Skill project as a target, choosing the following options:
+
+- **? Choose a modeling stack for your skill:**  `Interaction Model`
+- **? Choose the programming language you will use to code your skill:**  `Python`
+- **? Choose a method to host your skill's backend resources:**  `Alexa Hosted`
+
+>*⚠️ If you don't choose the specified options on the New Skill Wizard, the process could fail as this template is made to run an Interaction Model skill in Python, while the Makefile method currently only supports Alexa Hosted skills.*
+
+The skill will start being created:
+```
+🎯 Creating a new hosted skill target
+
+Please follow the wizard to start your Alexa skill project ->
+? Choose a modeling stack for your skill:  Interaction Model
+  The Interaction Model stack enables you to define the user interactions with a combination of utterances, intents, and slots.
+? Choose the programming language you will use to code your skill:  Python
+? Choose a method to host your skill's backend resources:  Alexa-hosted skills
+  Host your skill code by Alexa (free).
+? Choose the default region for your skill:  eu-west-1
+? Please type in your skill name:  gemini flash
+? Please type in your folder name for the skill project (alphanumeric):  geminiflash
+⠧ Creating your Alexa hosted skill. It will take about a minute.
```

Comment:
given the stability with Alexa right now we perhaps should include some notes around this. Not sure what we should write though :/

Reply:
Good idea, added a note below ;)